test/web-service/__snapshots__/web-service.test.ts.snap

```diff
@@ -513,7 +543,6 @@ exports[`WebService > Props > All the props 1`] = `
         "alb.ingress.kubernetes.io/success-codes": "200,303",
         "alb.ingress.kubernetes.io/tags": "service=my-app,instance=web,environment=test",
         "alb.ingress.kubernetes.io/target-type": "ip",
-        "external-dns.alpha.kubernetes.io/hostname": "api.example.com",
```

Comment:
As per included fix in this PR - we don't want to set external DNS on canary Ingress, as it would lead to conflicting records.